### DISCUSSION TITLE
Add flag to provide custom repo/git-remote to publish the release on

### DISCUSSION THREAD

**Discussion by erwinvaneyk** [goreleaser/goreleaser#1432] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
I have a project which has multiple upstreams/github repositories. Right now goreleaser always tries to publish on the repo that is in the `origin` remote

**Describe the solution you'd like**
Ideally I would like a flag `--remote=alternativeRemote` or `--repo=myOrg/alternativeRepo` that will make goreleaser publish the release to that repository.

**Describe alternatives you've considered**
It is a small quality-of-life improvement that right now can be fixed by manually altering your git-remotes, so I can understand if this is beyond the scope of this project. 

**Additional context**
```
version: 0.131.1
commit: dec22cf0c8a6464390ea26b42086d242e396ecda
built at: 2020-04-03T02:30:00Z
built by: goreleaser
```

**Comment by caarlos0** [comment, marked as answer]
I think the release customization solves this already:

```yaml
release:
  github:
    owner: user
    name: repo
```

https://goreleaser.com/customization/#Release

have you tried that?

<blockquote><img src="https://goreleaser.com/card.png" width="48" align="right"><div><strong><a href="https://goreleaser.com/">GoReleaser</a></strong></div><div>Deliver Go binaries as fast and easily as possible</div></blockquote>

**Comment by erwinvaneyk** [comment]
Hey @caarlos0 - thanks for the fast response! I saw it, but it's not the ideal to me either, since it turns the repository into a dirty state, which makes it about as much effort as swapping the origin remote. 

But, I can see that the feature I am proposing could be considered to minimal of an improvement, not worth the additional flag.

**Reply by caarlos0** [reply]
ok, I'll close this then.
